.github/workflows/scripts/linux/generate-cmake.sh

```diff
@@ -10,17 +10,24 @@ else
   export CXX=clang++
 fi
 
-if [ "${PLATFORM}" = "x86" ]; then
-  ADDITIONAL_CMAKE_ARGS="$ADDITIONAL_CMAKE_ARGS -DCMAKE_TOOLCHAIN_FILE=cmake/linux-compiler-i386-multilib.cmake"
+if [ "${PLATFORM}" = x86 ]; then
+  ADDITIONAL_CMAKE_ARGS="$ADDITIONAL_CMAKE_ARGS" 
```

Comment:
Any reason for separating the toolchain file out of `ADDITIONAL_CMAKE_ARGS`?

Reply:
`Codacy` complains if the variable is not double-quoted, and the entire CMAKE command will fail if the value is non-empty and quoted.

https://github.com/PCSX2/pcsx2/blob/c5505168d32e8946c39c58a44af851fa6a37756e/.github/workflows/scripts/linux/generate-cmake.sh#L30-L31

By separating it into two lines, double-quotes work whether the value is empty or not.